From 4a9029f560d4f04ab6e411ebf636e76dd3de4a21 Mon Sep 17 00:00:00 2001
From: Kei Okada <k-okada@jsk.t.u-tokyo.ac.jp>
Date: Thu, 15 May 2014 18:54:06 +0900
Subject: [PATCH] add ros-groovy-pr2-controllrs to Depends

---
 debian/control.em |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/debian/control.em b/debian/control.em
index 35e0893..c82a91e 100644
--- a/debian/control.em
+++ b/debian/control.em
@@ -2,11 +2,11 @@ Source: @(Package)
 Section: misc
 Priority: extra
 Maintainer: @(Maintainer)
-Build-Depends: debhelper (>= 7.0.50~), @(', '.join(BuildDepends))
+Build-Depends: debhelper (>= 7.0.50~), @(', '.join(BuildDepends)), ros-groovy-pr2-controllers
 Homepage: @(Homepage)
 Standards-Version: 3.9.2
 
 Package: @(Package)
 Architecture: any
-Depends: ${shlibs:Depends}, ${misc:Depends}, @(', '.join(Depends))
+Depends: ${shlibs:Depends}, ${misc:Depends}, @(', '.join(Depends)), ros-groovy-pr2-controllers
 Description: @(Description)
-- 
1.7.9.5

