From b2f86c8de047d214150477d549233b39819a6262 Mon Sep 17 00:00:00 2001
From: Kei Okada <k-okada@jsk.t.u-tokyo.ac.jp>
Date: Mon, 23 Jun 2014 16:41:19 +0900
Subject: [PATCH] add ros-groovy-pr2-controllrs to Depends

---
 debian/control.em |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/debian/control.em b/debian/control.em
index 8b9cd67..1930620 100644
--- a/debian/control.em
+++ b/debian/control.em
@@ -2,11 +2,11 @@ Source: @(Package)
 Section: misc
 Priority: extra
 Maintainer: @(Maintainer)
-Build-Depends: debhelper (>= @(debhelper_version).0.0), @(', '.join(BuildDepends))
+Build-Depends: debhelper (>= @(debhelper_version).0.0), @(', '.join(BuildDepends)), ros-groovy-pr2-controllers
 Homepage: @(Homepage)
 Standards-Version: 3.9.2
 
 Package: @(Package)
 Architecture: any
-Depends: ${shlibs:Depends}, ${misc:Depends}, @(', '.join(Depends))
+Depends: ${shlibs:Depends}, ${misc:Depends}, @(', '.join(Depends)), ros-groovy-pr2-controllers
 Description: @(Description)
-- 
1.7.9.5

