From d39349e40ea536beef3f3be7a73284d6e5894b41 Mon Sep 17 00:00:00 2001
From: lokada <lokada@jenkins.jsk.imi.i.u-tokyo.ac.jp>
Date: Fri, 16 May 2014 01:00:52 +0900
Subject: [PATCH 2/2] remove build depend to pr2-controllers

---
 debian/control.em |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/control.em b/debian/control.em
index c82a91e..16270e6 100644
--- a/debian/control.em
+++ b/debian/control.em
@@ -2,7 +2,7 @@ Source: @(Package)
 Section: misc
 Priority: extra
 Maintainer: @(Maintainer)
-Build-Depends: debhelper (>= 7.0.50~), @(', '.join(BuildDepends)), ros-groovy-pr2-controllers
+Build-Depends: debhelper (>= 7.0.50~), @(', '.join(BuildDepends))
 Homepage: @(Homepage)
 Standards-Version: 3.9.2
 
-- 
1.7.9.5

